packages/compiler/src/shared/types.ts

```diff
@@ -106,6 +106,16 @@ export interface TSXResult {
   code: string;
   map: SourceMap;
   diagnostics: DiagnosticMessage[];
+  metaRanges: {
+    frontmatter: {
+      start: number;
+      end: number;
+    };
```

Comment:
Are these always present even when we don't have the frontmatter?

Reply:
Yes, because there's still a location for where the frontmatter would be, after the prefix, but before the template, it just has the same value for both the start and the end (since it has no content)